source/twitter.css

```diff
@@ -0,0 +1,10 @@
+/* Twitter */
+
+.account-group strong,
+img.avatar.js-action-profile-avatar,
+.ProfileCanopy-headerBg img,
+h1.ProfileHeaderCard-name,
+.js-retweet-text b,
+img.ProfileAvatar-image {
+  display: none;
```

Comment:
Thanks for the feedback, @dawsbot! That suggestion seems to work much better than `display: hidden`.